src/measures.cc

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+#include "measures.h"
+
+#include <absl/strings/string_view.h>
+#include <opencensus/stats/stats.h>
+
+namespace google {
+
+namespace {
+
+constexpr char kUnitBytes[] = "By";
+constexpr char kCount[] = "1";
+
+} // namespace
+
+ABSL_CONST_INIT const absl::string_view
```

Comment:
Btw, why does this use string_view?

Reply:
opencensus measure registration (opencensus::stats::<MeasureT>::Register) uses string_view. 

That said, kCount is not absl::string_view either. I should make them consistent.